src/libraries/System.Globalization/tests/System/Globalization/RegionInfoTests.cs

```diff
@@ -90,7 +90,8 @@ public void DisplayName(string name, string expected)
             }
         }
 
-        [Theory]
+        // RegionInfo.NativeName is not supported by Browser's ICU
+        [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsNotBrowser))]
```

Comment:
I just tried this out and the test seems to work here?

Reply:
@akoeplinger it won't work with the next version of ICU.dat (-1.2Mb)

Reply:
Use `PlatformSpecificAttribute`?